app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -89,13 +91,18 @@ class ChatActivity: AppCompatActivity(), IChatView {
         }
         registerReceiver(networkStateReceiver, IntentFilter(ConnectivityManager.CONNECTIVITY_ACTION))
         recognizer = SpeechRecognizer.createSpeechRecognizer(applicationContext)
+
+        if(example != null && !example.isEmpty()) {
```

Comment:
you can use it in onResume method because nondelivered messages get cleared in onResume so sometimes messages get cleared before app sends it to the server.